### PR TITLE
params: document --targets option

### PR DESCRIPTION
**Pull request by pared**
Related to iterative/dvc#4446 and iterative/dvc#5122.

Introduce description for new `params` option: `--targets`.

### CODE REVIEW

File: content/docs/command-reference/params/diff.md [hunk 1]
```diff
@@ -35,6 +36,20 @@ itself does not ascribe any specific meaning for these values.
 
 ## Options
 
+- `--targets` - Limit command scope to these params files. When specifying
```
Comment (shcheklein) [inline content/docs/command-reference/params/diff.md:39]: Limit -> limit

Comment (jorgeorpinel) [reply content/docs/command-reference/params/diff.md:39]:
```suggestion
- `--targets <paths>` - limit command scope to these params files. When specifying
```

**Review by jorgeorpinel — APPROVED** [review]

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
Hey @pared will iterative/dvc/pull/5122 be part of 1.x or 2.x? To know whether to put this in our `v1` branch too or not. Thanks

**Comment by shcheklein** [comment]
@jorgeorpinel we have to track, ask the team about the backport PRs and do backport on the DVC.org end accordingly. They are designated as far as I know with a label?

**Comment by jorgeorpinel** [comment]
OK. There's no special labels in iterative/dvc/issues/4446 or iterative/dvc/pull/5122 though. And that PR isn't merged yet.

**Comment by jorgeorpinel** [comment]
> There's no special labels in iterative/dvc/issues/4446 or iterative/dvc/pull/5122

So I will assume that this is NOT 2.0 and can go into our `v1` branch.

**Comment by jorgeorpinel** [comment]
✔️ Backported to `v1` in 28f46ee4.

**Pull request merged**